first draft of Training Logistics episode (#10)
carpentries/lessondev-trainer-training: episodes/operations.md

## Reading List

-[CLDT Episode: How We Operate](https://carpentries.github.io/lesson-development-training/19-operations.html)
- The [information on Trial Runs](https://carpentries.github.io/lesson-development-training/trial-runs.html)
  provided to trainees during CLDT.
- Familiarise yourself with [the list of open issues on the CLDT curriculum repository][cldt-issues], 
  especially those assigned to [the 'Beta Release' milestone][beta-milestone].
- The content of this episode below.

::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::

## Structure of Collaborative Lesson Development Training

Collaborative Lesson Development Training is taught in two parts,
with an extended break in between.
Trainees are asked to teach a "trial run" during this break:
a mini-workshop where they teach a part of the lesson they are developing
and gather feedback (directly and indirectly)
about how effective their content is for teaching the objective skills.

> ![Collaborative Lesson Development Training is designed to be taught over three days
> (six half days),
> with an extended break between the first two days ("part 1")
> and the last day ("part 2").

Reviewer: ```suggestion
![Collaborative Lesson Development Training is designed to be taught over six half days,
with an extended break between **part 1** (half days 1-4)
and **part 2** (half days 5-6)
If taught in-person, the schedule could be adjusted to teaching part 1 in 2 full days and following up with part 2 in 1 full day.
```

> The training in part 1 focusses on good practices in lesson design and development,
> and the concepts and skills needed to build a lesson website with The Carpentries lesson infrastructure.

Reviewer: ```suggestion
The training in part 1 focusses on good practices in lesson design and development,
and the concepts and skills needed to build a lesson website with The Carpentries lesson infrastructure.
Between part 1 and part 2 the participants schedule a session to trail parts of their lesson with a real (or mock) audience and collect feedback.
```

> Part 2 of the training begins with a reflective discussion of lesson deisgn and iterative development,

Reviewer: ```suggestion
Part 2 of the training begins with a reflective discussion of lesson design and iterative development,
```

Reviewer: ```suggestion
Part 2 of the training begins with a reflective discussion of the trial runs to encourage participants to think about lesson design and iterative development,
```
alt suggestion that brings together the trail runs part

Trainees will only be able to sign up to join a part 2 event 
after registering their Trial Run. 
(Although they can register the Trial Run in advance i.e. it will not need to have happened yet).

Like the system for scheduling trainings, 
the system/process for Trial Run registration is still under development.
In the training pilots, we used a relatively simple web form 
but we may want to incorporate this data collection into AMY,
to integrate it into the workflows for tracking trainee progress centrally.

## What do Trainees require for certification?

For a trainee to receive their Lesson Developer certificate,
they will be required to:

1. participate in both parts of Collaborative Lesson Development Training.
2. teach a trial run of part of their lesson, and register that trial run with The Carpentries.

:::::::::::::::::::::::::::::::::::::::::::::::::: discussion

## Discussion Questions

- This training is about lesson design and development skills, not teaching skills.
  How important do you think it is that 
  every member of a collaborative group should _have to_ teach during their trial run - 

>   as opposed to, say, one member teaching all of the groups' prepared episodes?

Reviewer: Should this question mention that the other developers would still be required to attend/support the trial run?

Author: I thought about this but decided to leave the ambiguity, hoping to prompt discussion around whether it would be acceptable for one person to teach the Trial Run on their own and report back to their collaborators, or would everyone need to be there e.g. as helpers? What if one person was missing but the other three could attend? etc.

Reviewer: That sounds like a great idea Toby!

- What questions do you have about how lesson development is done
  in The Carpentries community?
  E.g. The Carpentries Incubator, The Carpentries Lab, 
  the lesson life cycle, 
  the individual lesson programs of The Carpentries,
  Curriculum Advisors, Maintainers,
  and how all of these things fit together?
- What questions do you have about how Collaborative Lesson Development Training
  will be administered?

> - How confident are you feeling about teaching the training for the first time?

Reviewer: ```suggestion
- How confident are you feeling about teaching the training for the first time? What support do you need to feel confident in teaching the training?
```

## Reading List

> - The [_How We Operate_](https://carpentries.github.io/lesson-development-training/19-operations.html) episode

Reviewer: ```suggestion
-[CLDT Episode: How We Operate](https://carpentries.github.io/lesson-development-training/19-operations.html)
```
Matching other PRs

> - The [_How We Operate_](https://carpentries.github.io/lesson-development-training/19-operations.html) episode
>   of the CLDT curriculum.

Reviewer: ```suggestion
-[CLDT Episode: How We Operate](https://carpentries.github.io/lesson-development-training/19-operations.html)
```
Matching other PRs

- The [information on Trial Runs](https://carpentries.github.io/lesson-development-training/trial-runs.html)
  provided to trainees during CLDT.
- Familiarise yourself with [the list of open issues on the CLDT curriculum repository][cldt-issues], 
  especially those assigned to [the 'Beta Release' milestone][beta-milestone].
- The content of this episode below.

::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::

## Structure of Collaborative Lesson Development Training

Collaborative Lesson Development Training is taught in two parts,
with an extended break in between.
Trainees are asked to teach a "trial run" during this break:
a mini-workshop where they teach a part of the lesson they are developing
and gather feedback (directly and indirectly)

> about how effective their content is for teaching the objective skills.

Reviewer: ```suggestion
about how effective their content is for teaching the objectives.
```
